openmetadata-ui/src/main/resources/ui/src/components/BotListV1/BotListV1.component.tsx

```diff
@@ -90,8 +90,15 @@ const BotListV1 = ({ showDeleted }: BotListV1Props) => {
         width: 90,
         render: (_, record) => (
           <Space align="center" size={8}>
-            <Tooltip placement="bottom" title="Delete">
+            <Tooltip
+              placement="bottom"
+              title={
+                permissions.Delete
+                  ? 'Delete'
+                  : 'You do not have permissions to perform this action.'
+              }>
```

Comment:
We can create `PermissionContainer` which can accept hasPermission & children
If no permission found we can disabled it and wrap around a tooltip

Reply:
Sure.